src/app/pages/offline-map/offline-map.page.html

```diff
@@ -32,7 +32,13 @@ <h1>Nedlastning av kartpakker</h1>
       >
     </div>
   </div> 
-
+  
+  <ion-item *ngIf="userSettings">
```

Comment:
Burde ikke denne infoen blitt vist et annet sted enn inne på menyen for offline-kart?
Siden dette er snakk om timeout på alle kartfliser og ikke noe som har med offlinekart å gjøre?

Reply:
Timeout på online-fliser settes kun hvis vi har offline-kart tilgjengelig